runtime/queries/html/textobjects.scm

```diff
@@ -0,0 +1,11 @@
+(element
+  (start_tag)
+  (_)* @xml_element.inside
+  (end_tag)
+. ) @xml_element.around
```

Comment:
We may want to duplicate this stanza for `style_element` and `script_element` since those are separate rules in the grammar. They both need `end_tag`s though so they won't need the `?` suggestion above

Reply:
Added these